pages/support.html

```diff
@@ -14,42 +14,51 @@ <H1>{% trans "Community Support" %}</H1>
     </p>
   </div>
+ 
   <div style="clear: both;"></div>
-
+  
   <div class="halfbox_left">
-    <h2>{% trans "Community Forums" %}</h2>
+    <h2>{% trans "User Manual" %}</h2>
     <p>
-      {% blocktrans with forums_link="/forums" %}Have a Mixxx-related question? Something other DJs can help you with? Ask in the <a href="{{ forums_link }}">Mixxx Community Forums</a>.{% endblocktrans %}
+      {% blocktrans with manual_latest_link="/manual/latest" manual_beginner_link="/manual/latest/chapters/djing_with_mixxx.html" %}The <a href="{{ manual_latest_link }}">Mixxx Manual</a> provides an overview of Mixxx's user interface, information about <b>configuration</b> and hardware, and much more. It also contains useful information for <b>beginners</b> under <a href="{{ manual_beginner_link }}">DJing with Mixxx</a>.{% endblocktrans %}
     </p>
   </div>
 
   <div class="halfbox_right">
     <h2>{% trans "Zulip Chat" %}</h2>
     <p>
-      {% blocktrans with zulip_web_link="https://mixxx.zulipchat.com/" zulip_app_link="https://zulipchat.com/apps/" %}<a href="{{ zulip_web_link }}">Mixxx's Zulip chat instance</a> can be used for support too. If you don't need support, Zulip is still a fun place to hang out with the developers and meet some fellow DJs. Join the discussion <a href="{{ zulip_web_link }} ">on the web</a> or download the <a href="{{ zulip_app_link }}">Zulip app</a> and configure it to use the server mixxx.zulipchat.com{% endblocktrans %}
+      {% blocktrans with zulip_web_link="https://mixxx.zulipchat.com/" zulip_app_link="https://zulipchat.com/apps/" %}<a href="{{ zulip_web_link }}">Mixxx's Zulip chat instance</a> is a fun place to hang out with the developers and meet some fellow DJs. Here, the Mixxx team discusses upcoming changes and programming related stuff. It can be used for support too. Join the discussion <a href="{{ zulip_web_link }} ">on the web</a> or download the <a href="{{ zulip_app_link }}">Zulip app</a> and configure it to use the server mixxx.zulipchat.com. {% endblocktrans %}
     </p>
-  </div>
+  </div> 
 
   <div style="clear: both;"></div>
 
+  <div class="halfbox_left">
+    <h2>{% trans "Mailing List" %}</h2>
+    <p>
+      {% blocktrans with mailing_list_link="https://lists.sourceforge.net/lists/listinfo/mixxx-devel" sourceforge_archive_link="http://sourceforge.net/mailarchive/forum.php?forum=mixxx-devel" gmane_archive_link="http://news.gmane.org/gmane.comp.multimedia.mixxx.devel" %}The <a href="{{ mailing_list_link}} ">mixxx-devel</a> mailing list (<a href="{{ sourceforge_archive_link }}">SourceForge</a> and <a href="{{ gmane_archive_link }}">Gmane</a> archives) is now a low traffic anouncment channel. Subscribe to this list, if you like to keep informed about new releases and other topics. {% endblocktrans %}
```

Comment:
I would change the text to 
> is now a low-traffic, mostly announcement channel. Subscribe if you would like to be informed about new releases and other topics.

Reply:
please note the additional 'e' and 'n' in announcement.